lib/Controller/ChatController.php

```diff
@@ -264,6 +264,9 @@ public function receiveMessages($token, $lookIntoFuture, $limit = 100, $lastKnow
 		$limit = min(200, $limit);
 		$timeout = min(60, $timeout);
```

Comment:
Maybe the `$timeout` should be limited to make sure it's not larger than the ping timeout. Otherwise clients with a too large timeout could get removed from the room due to ping inactivity while waiting for chat messages.

Reply:
fixed